jetty-servlets/src/test/java/org/eclipse/jetty/server/handler/gzip/GzipDefaultTest.java

```diff
@@ -468,6 +468,7 @@ public void testGzipCompressedByContentTypeWithEncoding() throws Exception
         tester.setContentServlet(HttpContentTypeWithEncoding.class);
         tester.getGzipHandler().addIncludedMimeTypes("text/plain");
         tester.getGzipHandler().setExcludedAgentPatterns();
+        tester.getGzipHandler().setMinGzipSize(16);
```

Comment:
I bet if you had a testcase in this actually producing 17 bytes of input content you'd see the break even issue.